pymmh3.py

```diff
@@ -14,18 +14,17 @@
 
 This module is written to have the same format as mmh3 python package found here for simple conversions:
 
-https://pypi.python.org/pypi/mmh3/2.0 
+https://pypi.python.org/pypi/mmh3/2.3.1
 '''
 
-import sys
-
-if (sys.version_info > (3, 0)):
+import sys as _sys
```

Comment:
Is that usually something python-folks care about? :) Personally I just find it to clutter the code for no real gain.